test/src/lib/sub-bots/sub-bots.js

```diff
@@ -60,48 +73,51 @@ describe('sub-bots/sub-bots.js', function() {
         });
     });
 
-    describe('test passBot rejection due to disabled feature', function() {
-        let result = false;
-
-        before(function(done) {
-            result = false;
-            config.setProperty('subBots', false);
-            subBots.passBot()
-                .then(passed => {
-                    subBots.logout(passed);
-                    config.setProperty('subBots', original);
-                    done();
-                })
-                .catch(err => {
-                    result = err;
-                    config.setProperty('subBots', original);
-                    done();
-                });
-        });
-
-        it('should reject if feature disabled', function() {
-            result.should.not.be.false;
-            result.should.eql('The sub bot feature is disabled or there are no subBots on file');
-        });
-    });
+    // Test doesn't work as subbots object in the config is not the same object
```

Comment:
Firstly, omg y u no /* comment */ that's like a solid minute of commenting

Secondly, we'll issue this, probably just need to add a feature switch or something

Reply:
Srry :| I just select a block of code and then press cmd + / (windows ctrl + /) and it will make all lines into comments (I am very lazy). :)

Reply:
Ah I see, I was not aware of such shortcut magic